frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -5,13 +5,14 @@ import * as React from 'react'
 import { Dataset } from '../../constants/datatypes'
 import { Store, withStore } from '../../services/store'
 import { download } from '../../utils/download'
+import { AddDatasetServiceAreaSelector } from '../AddDatatsetServiceAreaSelector/AddDatasetServiceAreaSelector'
```

Comment:
Not sure I like `ServiceAreaSelector` here but dont really have a better idea...